### ISSUE TITLE
Constraint on Conv2D

### ISSUE THREAD

**Issue by bernardohenz** [fchollet/keras#3628]
How can I set the maximum norm for **each weight** on the convolution kernel?
I've tried to use the `MaxNorm`, but I don't know which axis should I specify. If I need to create my own Constraint, how should I code it? Something like the following code should work?:

``` python
'''
# Arguments
     m: the maximum norm for the incoming weights.
'''
def __call__(self, p):
        norms = K.sqrt(K.square(p))
        desired = K.clip(norms, 0, self.m)
        p = p * (desired / (K.epsilon() + norms))
return p
```

Please make sure that the boxes below are checked before you submit your issue. Thank you!
- [x] Check that you are up-to-date with the master branch of Keras. You can update with:
  pip install git+git://github.com/fchollet/keras.git --upgrade --no-deps
- [X] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
  pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps
- [X] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).
